content/en/agent/guide/datadog-disaster-recovery.md

```diff
@@ -0,0 +1,284 @@
+4. [you are ready to test the failover process](#when-you-are-ready-to-test-the-failover-process)
+
+#### when you are ready to configure Datadog Disaster Recovery: 
+{{% collapse-content title=" 1. Create your secondary Datadog organization" level="h5" %}}
+Identify which site your primary organization is on by matching your Datadog website URL to the [`SITE URL`][4] in the table. Then, select a secondary site for your DDR organization.
+
+{{< img src="getting_started/site/site.png" alt="The site URL in your browser tab" style="width:40%" >}}
+
+| Site    | Site URL                    | Site Parameter      | Location |
+|---------|-----------------------------|---------------------|----------|
+| US1     | `https://app.datadoghq.com` | `datadoghq.com`     | US       |
+| US3     | `https://us3.datadoghq.com` | `us3.datadoghq.com` | US       |
+| US5     | `https://us5.datadoghq.com` | `us5.datadoghq.com` | US       |
+| EU1     | `https://app.datadoghq.eu`  | `datadoghq.eu`      | EU (Germany) |
+| AP1     | `https://ap1.datadoghq.com` | `ap1.datadoghq.com` | Japan |
+
+For example, if you are hosted in [US1](https://app.datadoghq.com), you may choose to select another Datadog site to ensure observability continuity in the event of a regional disaster. All Datadog sites are geographically separated.
+
+Contact your [Customer Success Manager](mailto:success@datadoghq.com) or [Datadog Support](https://www.datadoghq.com/support/) to help you select the secondary Datadog site and configure your new organization to be your secondary failover organization.
+
+If you're also sending telemetry to Datadog using cloud provider integrations, you must add your cloud provider accounts in the secondary org. 
+
+Datadog does not use cloud providers to receive telemetry data while the secondary site is passive.
+
+**Note**: Datadog can set this up for you if you'd prefer. 
+{{% /collapse-content %}}
+
+
+{{% collapse-content title=" 2. Contact Datadog to share your new organization" level="h5" %}}
+Share your organization name with your Datadog [Customer Success Manager](mailto:success@datadoghq.com) so they can configure your new organization to be your secondary failover organization.<br><br>
+**Note**: This organization appears in your Datadog billing hierarchy, but all usage and cost associated is _not_  billed during the private beta.
+{{% /collapse-content %}} <br>
+
+
+
+#### when Datadog confirms your new organization as your secondary failover organization
+{{% collapse-content title=" 3. Confirm the public IDs of your orgs" level="h5" %}}
+After the Datadog team has completed the configuration of the designated orgs, Datadog will share with you the public IDs of the primary org and the DDR org. You can verify these IDs using the cURL commands from the Datadog [public API endpoint][8]:  
+
+``` shell
+# Run this command to verify the Public ID for your primary site replacing the `<DD-SITE-PARAMETER>` with the correct value. 
```

Comment:
Is there a reason we refer to primary and DDR `orgs` above, but `sites` in these code samples?

Also, I'm not 100% on how necessary these steps are. Is this to check to ensure the IDs Support sent are correct...? If this isn't necessary, I'd just link to the API page with these and say that users can test them if they want to, and save them some work if they feel confident in the IDs they received.